src/palace.pyx

```diff
@@ -2307,11 +2307,11 @@ cdef class ReverbEffect(BaseEffect):
         self.slot.apply_effect(self.impl)
 
     @property
-    def decay_hf_limit(self) -> int:
+    def decay_hf_limit(self) -> bool:
         return self.properties.decay_hf_limit
 
     @decay_hf_limit.setter
-    def decay_hf_limit(self, value: int) -> None:
+    def decay_hf_limit(self, value: bool) -> None:
```

Comment:
Genuinely asking, how is typing enforced by these hints, i.e. what's `type(ReverbEffect().decay_hf_limit)` initially and after, say, setting `.decay_hf_limit = []`?  IMHO it's OK to have boolean variable to be set to anything, long as the functionality works.

Though this one belong to another PR.